src/index.tsx

```diff
@@ -130,7 +135,7 @@ export const BatteryGauge: FC<Props> = ({
     enabled: animated,
   });
   const chargingValue = useCounterAnimation({
-    startValue: 1,
+    startValue: chargingStartValue,
```

Comment:
```suggestion
    startValue: showChargedReading? value : 1,
```
Right now i can think of these two conditions only. However, let us go with flexible `chargingStartValue` for now.